# TaskArray.msg - Array of tasks
std_msgs/Header header
decentralized_auction_mm/Task[] tasks